### PR TITLE
0.3.0

### PR DESCRIPTION
**Pull request by BenFradet**
- [x] re-enable https tests
- [x] changelog date

### CODE REVIEW

**Comment by alexanderdean** [comment]
I think @chuwy can review this one? Ben please make sure there is a Trello card for this...

File: src/test/scala/com.snowplowanalytics.weather/providers/openweather/ServerSpec.scala [hunk 1]
```diff
@@ -76,12 +69,4 @@ class ServerSpec
       case e: WeatherError => e.toString must beEqualTo("OpenWeatherMap AuthorizationError$ Check your API key")
     }
   }
-
-  def e4 = {
```
Comment (chuwy) [inline src/test/scala/com.snowplowanalytics.weather/providers/openweather/ServerSpec.scala]: Why this has been deleted?

File: src/main/scala/com.snowplowanalytics/weather/providers/openweather/HttpTransport.scala [hunk 2]
```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2015 Snowplow Analytics Ltd. All rights reserved.
```
Comment (chuwy) [inline src/main/scala/com.snowplowanalytics/weather/providers/openweather/HttpTransport.scala]: 2017

File: src/test/scala/com.snowplowanalytics.weather/providers/openweather/ServerSpec.scala (hunk 1 above)
Comment (BenFradet) [inline src/test/scala/com.snowplowanalytics.weather/providers/openweather/ServerSpec.scala]: because now it always give back unauthorized

Comment (chuwy) [inline src/test/scala/com.snowplowanalytics.weather/providers/openweather/ServerSpec.scala]: Why is that?

**Comment by chuwy** [comment]
Also, I see unimplemented issue in milestone https://github.com/snowplow/scala-weather/issues/18

File: src/test/scala/com.snowplowanalytics.weather/providers/openweather/ServerSpec.scala (hunk 1 above)
Comment (BenFradet) [inline src/test/scala/com.snowplowanalytics.weather/providers/openweather/ServerSpec.scala]: I don't know they must have stopped caring about giving useful error messages.

**Comment by BenFradet** [comment]
Welp, originally it was just to make the ssc assembly possible.
But since, as you say, we might not work on this codebase for a while we might aswell make the necessary changes.

File: src/test/scala/com.snowplowanalytics.weather/providers/openweather/ServerSpec.scala (hunk 1 above)
Comment (chuwy) [inline src/test/scala/com.snowplowanalytics.weather/providers/openweather/ServerSpec.scala]: Ok, I noticed OWM changed its behavior many times.

**Comment by chuwy** [comment]
Up to you! (CI/CD unfortunately is not optional)

**Comment by alexanderdean** [comment]
> But since, as you say, we might not work on this codebase for a while we might aswell make the necessary changes.

Agree, thanks guys

**Comment by BenFradet** [comment]
@alexanderdean @chuwy should we block the release over the testing of https / getting the key?

also @chuwy, could you have another look?

File: project/build.properties [hunk 3]
```diff
@@ -1 +1 @@
-sbt.version=0.13.9
+sbt.version=0.13.15
```
Comment (chuwy) [inline project/build.properties]: 0.13.16 was released today.

File: CHANGELOG [hunk 4]
```diff
@@ -1,3 +1,19 @@
+Version 0.3.0 (2017-07-27)
```
Comment (chuwy) [inline CHANGELOG]: Don't forget to bump date.

File: project/Dependencies.scala [hunk 5]
```diff
@@ -20,65 +20,33 @@ object Dependencies {
     // For scala-util
```
Comment (chuwy) [inline project/Dependencies.scala]: Doesn't seem we use/used scala-util here.

**Comment by BenFradet** [comment]
@chuwy waiting for a bit sounds sensible to me, this isn't really needed until we're close to merging r91.

**Comment by alexanderdean** [comment]
Sounds like a plan!

File: project/Dependencies.scala [hunk 6]
```diff
@@ -16,69 +16,34 @@ import Keys._
...
-    val akkaStreams   = "2.0.2"
-
-    object collUtil {
-      val _210      = "6.3.4"
-      val _211      = "6.29.0"  // Should work on 2.10 too
-    }
-
+    val json4s   = "3.2.11"
+    val scalaj   = "2.3.0"
+    val collUtil = "6.34.0"
```
Comment (chuwy) [inline project/Dependencies.scala]: If we can bump `collUtil` to `6.39.0` then we will be able to publish scala-weather for Scala 2.12. Json4s for 2.12 has been published few days ago https://github.com/json4s/json4s/releases/tag/v3.2.11_2.12

Comment (BenFradet) [inline project/Dependencies.scala]: good idea, done :+1:

Comment (chuwy) [inline project/Dependencies.scala]: Awesome! Moving towards future.

**Comment by BenFradet** [comment]
we're ready to go :+1: , @chuwy if you want to have a final look

**Comment by chuwy** [comment]
It's good to go!

**Pull request merged**